pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+At a pH where 90.0% of the molecules are in microstate B and 10.0% of molecules are in state A  `ln(fractional microstate A population) = ln(0.100/(0.100+0.900)) = -2.30E+00`.  
+- If your estimate of `fractional microstate population` is 0, thus `ln(fractional microstate population) = ln(0)`, report as `-infinity`.
+- Do not report SEM in this submission type.
+- For pH value or microstates which you don't have any estimates for, leave the csv table cell empty. 
+
+#### Type III - macroscopic pKas
+Predicting the value of  macroscopic pKas based between 2 and 12.
+- Fill one `typeIII_macroscopic_pKas.csv` template file for all predicted molecules.
+- Report as many macroscopic pKas based on your predictions.
+- Report pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reproted with 2 decimals. (e.g. 1.02).
+
+## A warning about enumerated microstates and requesting missing microstates
+A list of microstates and microstate IDs were generated for each molecule to aid parsing of submissions. Enumerated list of microstates was not created with the intend to guide computational predictions. It is possible that some relevant microscopic species are missing from these lists. If your predicted structure is not already included in the microstates list, contact Mehtap Isik (mehtap.isik@choderalab.org). Please send us 2D structure depiction and canonical isomeric SMILES of your proposed microstate. We will assign a unique microstate ID and include it in the analysis.  
+
+Please do not create a microstate ID yourself. It is important that challenge organizers assign unique microstate IDs and keep track.  
+
+## Submission of multiple predictions
+
+Some participants use SAMPL to help evaluate various computational methods. To accommodate this, multiple prediction sets from a single research group or company are allowed, even for same type of predictions if they are made by different methods.
+
+## Uploading your predictions
+
+D3R is currently outfitting the SAMPL6 page with the ability to accept your uploaded predictions. As soon as this is ready, you may upload your predictions. If you want to upload more than one type predictions (see description of three pKa prediction types above) or different set of predictions (same type but generated by different methods), each must be uploaded as a separate file. Please use the template provided, as the predictions will be parsed and analyzed with automated scripts. Please include predictions related all molecules with same method and same submission type in one file. 
+
+We encourage submitting predictions in all three formats and for all 24 molecules when possible. Incomplete submissions - such as for a subset of compounds - will also be accepted, but will not necessarily be evaluated together with the rest of the submissions. However, we would emphasize that omission of SEM estimates will not cause a submission to be regarded as incomplete.
```

Comment:
"as incomplete, though we highly encourage including such estimates." If you don't encourage people to do error estimates they almost certainly won't.

Reply:
Done.